src/cli/Vaults.ts

```diff
@@ -47,6 +77,50 @@ function makeAddVaultCommand() {
     );
 }
 
+function makeCloneVaultCommand() {
+  return new commander.Command('clone')
+    .description('clone a vault from a peer')
```

Comment:
In our application, should we really be differentiating pull from clone? Can we merge these 2 actions together, so that pull clones if it doesn't exist. And pulls if it does exist.

Reply:
certainly, they can be separated at the implementation level (because there is a whole lot of logic in cloning a vault that isn't in pulling a vault) but abstracted into just one command for the user.

Reply:
Cloning has been replaced with just an expanded pull command that checks if the vault already exists and pulls, if not it then clones. Point is, this is no longer presented to the user as two things.